client/src/main/java/org/apache/rocketmq/client/consumer/DefaultMQPushConsumer.java

```diff
@@ -461,7 +461,11 @@ public void start() throws MQClientException {
      */
     @Override
     public void shutdown() {
-        this.defaultMQPushConsumerImpl.shutdown();
+        shutdown(0);
+    }
+
+    public void shutdown(long awaitTerminateMillis) {
```

Comment:
Since `MQPushConsumer` only has the method interface for `shutdown( )`, so we could consider pull `shutdown(long awaitTerminateMillis)` up to the parent interface or make `shutdown(long awaitTerminateMillis)` private and give a default input(non-zero) in `shutdown()`.